backend/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "simple-report-api",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "lint-staged": "^10.5.4"
+  },
+  "scripts": {
+    "precommit": "lint-staged"
```

Comment:
Couldn't the precommit hook here just be `./gradlew spotlessApply`? I don't think we're getting any value from the dev dependencies here.